datatypes/src/jvmTest/kotlin/DistinguishedNameTest.kt

```diff
@@ -0,0 +1,36 @@
+import at.asitplus.crypto.datatypes.asn1.ObjectIdentifier
+import at.asitplus.crypto.datatypes.pki.DistinguishedName
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+
+class DistinguishedNameTest : FreeSpec({
+    "DistinguishedName Test" - {
```

Comment:
maybe also augment this using property-based testing to check more arbitrary strings?